docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -107,7 +107,14 @@ To classify an image:
         dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
     ```
 
-4. Configure `g_engine_config` with classification model data to classify image. In the following example, TensorFlow Lite model is used and `data.tflite` and `label.txt` are in `<OwnDataPath>`. Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models):
+4. Configure `g_engine_config` with classification model data to classify image.
```

Comment:
Please add a prefix "**the**" before image. 

Configure `g_engine_config` with classification model data to classify the image.